src/RemediationRoleSaver.cpp

```diff
@@ -30,12 +30,36 @@ extern "C"
 #include <xccdf_benchmark.h>
 #include <xccdf_policy.h>
 #include <xccdf_session.h>
+#include <ds_rds_session.h>
```

Comment:
AFAIK ds_rds_session is only exposed in recent OpenSCAP, we should find the exact version and have a macro guard around this